agent/consul/acl_endpoint_test.go

```diff
@@ -4434,6 +4438,9 @@ func TestACLEndpoint_SecureIntroEndpoints_OnlyCreateLocalData(t *testing.T) {
 	})
 
 	t.Run("logout of remote token in remote dc", func(t *testing.T) {
+		// if the other test fails this one will to but will now not segfault
```

Comment:
One alternative that's more work, is to wrap each `t.Run` with a `require.True(t, t.Run(...))` that will force the overarching test to fail at the first subtest fail in a test like this where you  are wanting sequential stages for better test stage tracking, but really after the first stage fails the rest are meaningless.

Reply:
For visibility here:

The reason not to would be that then this would throw off the total number of tests run in CI so we couldn't see from run to run that the same number of tests are being executed.